visualization/dashboard/src/MLIDashboard/Controls/FeatureImportanceBar/FeatureImportanceBar.tsx

```diff
@@ -0,0 +1,159 @@
+                                    )}
+                                </div>
+                            </div>
+                        </div>
+            <AccessibleChart
+                plotlyProps={plotlyProps}
+                theme={getTheme() as any}
+                relayoutArg={relayoutArg as any}
+                onClickHandler={this.selectPointFromChart}
+            />
+        </div>);
+    }
+
+    private loadProps(): void {
+        setTimeout(() => {
+            const props = this.buildBarPlotlyProps();
+            this.setState({plotlyProps: props});
+            }, 1);
+    }
+
+    private buildBarPlotlyProps(): IPlotlyProperty {
+        const sortedIndexVector = this.props.sortArray;
+        const baseSeries = {
+            config: { displaylogo: false, responsive: true, displayModeBar: false } as Plotly.Config,
+            data: [],
+            layout: {
+                autosize: true,
+                dragmode: false,
+                barmode: 'group',
+                margin: {t: 10, r: 10, b: 30, l: 0},
+                hovermode: 'closest',
+                xaxis: {
+                    automargin: true,
+                    color: FabricStyles.chartAxisColor,
+                    tickfont: {
+                        family: "Roboto, Helvetica Neue, sans-serif",
+                        size: 11,
+                        color: FabricStyles.chartAxisColor,
+                    },
+                    showgrid: false
+                },
+                yaxis: {
+                    automargin: true,
+                    color: FabricStyles.chartAxisColor,
+                    tickfont: {
+                        family: "Roboto, Helvetica Neue, sans-serif",
+                        size: 11,
+                        color: FabricStyles.chartAxisColor,
+                    },
+                    zeroline: true,
+                    showgrid: true,
+                    gridcolor: "#e5e5e5"
+                },
+                showlegend: false
+            } as any
+        };
+
+        const x = sortedIndexVector.map((unused, index) => index);
+
+        this.props.unsortedSeries.forEach((series, seriesIndex) => {
+            baseSeries.data.push({
+                orientation: 'v',
+                type: 'bar',
+                name: series.name,
+                x,
+                y: sortedIndexVector.map(index => series.unsortedAggregateY[index]),
+                marker: {
+                    color: sortedIndexVector.map(index => (index === this.props.selectedFeatureIndex && seriesIndex === this.props.selectedSeriesIndex) ?
+                        FabricStyles.fabricColorPalette[series.colorIndex] : FabricStyles.fabricColorPalette[series.colorIndex])
+                }
+            } as any);
+            
+        });
+
+        const ticktext = sortedIndexVector.map(i =>this.props.unsortedX[i]);
```

Comment:
again missing space:
i =>this.props.unsortedX[i]
probably would be good to have linter